drivers/scheduler/k8s/specs/kubevirt-vm-pvc/kubevirt-app.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  creationTimestamp: 2018-07-04T15:03:08Z
+  generation: 1
+  labels:
+    kubevirt.io/os: linux
+  name: cirrosvm
+spec:
+  running: true
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        kubevirt.io/domain: cirrosvm
+    spec:
+      domain:
+        cpu:
+          cores: 2
+        devices:
+          disks:
+            - disk:
+                bus: virtio
+              name: disk0
+            - cdrom:
+                bus: sata
```

Comment:
Can we make it SCSI since it is widely used in VM world ?

Reply:
@trenukarya-px These are just the first cut of kubevirt VM specs that we have introduced.
These are just plain vanilla VMs with no apps running in it.
Once @vpinisetti-px creates specs for various types of VMs with various volume types, NICs etc., we will keeo adding it to our app list in Torpedo.
I am waiting for Venkat to test them manually and qualify them as golden templates, then add those specs to Torpedo.
Let me know your thoughts.